EHospital.Authorization/Helpers/AuthorizationOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace EHospital.Authorization.WebAPI
+{
+    using Microsoft.IdentityModel.Tokens;
+    using System.Text;
+
+    public class AuthorizationOptions
+    {
+        public const string ISSUER = "EHospital.AuthorizationServer";
+        public const string AUDIENCE = "http://localhost:44386/";
+        const string KEY = "bhiyrbgi893nixmbh47yhfbuj";
+        public const int LIFETIME = 30;
```

Comment:
Such thing as LIFETIME/KEY is good to put in config file. Key should be encoded.